README.md

```diff
@@ -103,6 +103,82 @@ t.Log(people)
 // stdout: [{Michał Matczuk [michal@scylladb.com]}]
 ```
 
+## Generating table metadata with schemagen
+
+Installation
+
+```bash
+go get -u "github.com/scylladb/gocqlx/v2/cmd/schemagen"
+```
+
+Usage:
+```bash
+$GOBIN/schemagen [flags]
+
+Flags:
+  -cluster string
+    	a comma-separated list of host:port tuples (default "127.0.0.1")
+  -keyspace string
+    	keyspace to inspect (required)
+  -output string
+    	the name of the folder to output to (default "models")
+  -pkgname string
+    	the name you wish to assign to your generated package (default "models") 
+```
+
+Example:
+
+Running the following command for `examples` [schema](https://github.com/Bobochka/gocqlx/blob/412899a9b2910d00f9f723ea304c3abef2263021/example_test.go#L77-L97): 
```

Comment:
The link points to your repo, I'm not sure we needed this perm link here, they tend to get obsolete.